tests/commons/glob/test_glob.py

```
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Any, Dict, List
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)